email-templates/201012.html

```diff
@@ -0,0 +1,58 @@
+<html>
+  <body>
+    <p>
+      <b class="TODO">Någon fräsig ingress</b>
+    </p>
+    <p>
+      <ul>
+        <li> Just nu går Hacktoberfest, ett månadslångt evenemang som
+             uppmärksammar open source. Om du gör fyra pull requests till
+             (medverkande) projekt på Github får du en Hacktoberfest-tröja!
+             (Eller så kan du välja att plantera ett träd.) Läs mer på
```

Comment:
Detta behöver inte vara en parantes. Du kan nog baka in det i den tidigare meningen. :)